### ISSUE TITLE
Learn WordPress Content Feedback - Broken Image Issue

### ISSUE THREAD

**Issue by huzaifaalmesbah** [WordPress/Learn#2346]
## Type of feedback
//content

## Description
The image fails to load, and only alt text is visible.

## Step-by-step reproduction instructions (optional)
1. Go to https://learn.wordpress.org/lesson/lets-talk-wordpress-login-security/
2. Scroll down to the bottom of the page
3. Locate How do I know if I’m logged in?
4. 
## Screenshots or screen recording (optional)
![Huzaifa-20240324133251](https://github.com/WordPress/Learn/assets/63150399/ee36d50b-d48c-46aa-b4f2-01dd815a6ef2)

## One More Next
1. Go to https://learn.wordpress.org/lesson/get-found-using-the-media-library/
2. Please scroll down a little bit on the page.
3. Try to Find :) 

## Screenshots or screen recording (optional)
![Huzaifa-20240324133338](https://github.com/WordPress/Learn/assets/63150399/604b32f4-4c53-4b36-a355-72d0eddf3b98)

**Comment by ironnysh** [comment]
# Feedback Validation Checklist:
- If this is reporting an issue, can you confirm/reproduce the issue? Yes: 
- What should happen next to apply the feedback?: Upload the screenshots provided below as replacements:

1. Link to lesson: https://learn.wordpress.org/lesson/lets-talk-wordpress-login-security/
2. Image file: ![Image](https://github.com/user-attachments/assets/b6f1660e-9756-4762-b109-3c42108731ce)

1. Link to lesson: https://learn.wordpress.org/lesson/get-found-using-the-media-library/
2. Image file: ![Image](https://github.com/user-attachments/assets/26cd3197-0e4e-4722-abaa-37879d1ae6c3)


@WordPress/learn-editors 

Note: I've recreated the screenshots based on the original `alt`:
1. `<img decoding="async" src="https://cdn-std.droplr.net/files/acc_1185858/gVaXs9" alt="The WordPress admin bar on a freshly installed Website. The links, "My WordPress Site", "Customize", the comment box, and "+ New" can be seen, as well as the words, "Howdy, adventuresings", which is the name of the user. The default blog post from a fresh WordPress installation is shown, posted on October 12th, 2021.">`
2. `<img decoding="async" src="https://cdn-std.droplr.net/files/acc_1185858/X53VUk" alt="The media library options when you click on an image. This picture highlights the alternative text, title, caption, file URL, and description fields.">`

**Comment by benazeer-ben** [comment]
Hi Team,

I have fixed the first image issue in this [lesson](https://learn.wordpress.org/lesson/lets-talk-wordpress-login-security/) using the suggested image.

For the second [lesson](https://learn.wordpress.org/lesson/get-found-using-the-media-library/), the reported image issue appears to be resolved, as the image is displaying correctly."

**Comment by devmuhib009** [comment]
Thanks @huzaifaalmesbah, @ironnysh and @benazeer-ben for taking care of the issue. 💕
